roles/openshift_facts/library/openshift_facts.py

```diff
@@ -323,6 +323,24 @@ def set_fluentd_facts_if_unset(facts):
             facts['common']['use_fluentd'] = use_fluentd
     return facts
 
+def set_master_schedulable(facts):
```

Comment:
Instead of set_master_schedulable, we might want to rename this schedulable and have it be a node fact....  Then we can default the value to false `if 'master' in facts`.  I would think we'd want that behavior by default for all installation types and not just online.